metron-platform/metron-parsers/src/main/java/org/apache/metron/parsers/utils/DateUtils.java

```diff
@@ -0,0 +1,78 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.metron.parsers.utils;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Various utilities for parsing and extracting dates
+ * 
+ */
+public class DateUtils {
+
+	public static List<SimpleDateFormat> DATE_FORMATS_CEF = new ArrayList<SimpleDateFormat>() {
+		{
+			// as per CEF Spec
+			add(new SimpleDateFormat("MMM dd HH:mm:ss.SSS zzz"));
+			add(new SimpleDateFormat("MMM dd HH:mm:ss.SSS"));
```

Comment:
What are the defaults in the case that no year and/or no timezone are provided? For example, in SysLogUtils we assume any datetime more than 4 days in the future is actually in the past. Would a parser config setting make sense to hold these default assumptions?